frontend/src/helpers/editActionDialogHelper.js

```diff
@@ -76,11 +76,18 @@ function isEmailFormEdited(
   let initialEmailCcArray = typeof initialEmailCc === "undefined" ? "" : initialEmailCc;
   let emailCcArray = typeof emailCc === "undefined" ? "" : emailCc;
 
-  // sorting the arrays before the comparison below
-  initialEmailToArray.sort();
-  emailToArray.sort();
-  initialEmailCcArray.sort();
-  emailCcArray.sort();
+  // sorting the arrays before the comparison below only if all To and Cc fields (initial and current) are defined
+  if (
```

Comment:
I think here you are basically checking if these arrays are not empty? That means you are checking for falsy-ness: https://developer.mozilla.org/en-US/docs/Glossary/Falsy

This should work
```
if (initialEmailTo && emailTo && initialEmailCc && emailCc) {
    initialEmailToArray.sort();
    emailToArray.sort();
    initialEmailCcArray.sort();
    emailCcArray.sort();
  }
```

But a few lines above you default these values to an empty string - shouldn't they be defaulted to an empty array?

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/247/commits/659978c13ea4134c6a4b88f92a7d06f8ea4f4a47 and commit #https://github.com/code-for-canada/project-thundercat/pull/247/commits/99bb7e2b94c8ad0005ba9ba3a2b62a045fb60e65